### DISCUSSION TITLE
Добавление screenshot тестов для безопасных изменений компонентов

### DISCUSSION THREAD

**Discussion by KirIIISolovyov** [chocofamilyme/choco-ui#114] [Ideas]
UiKit в большей степени готов, но переодически вносятся небольшие изменения. Т.к UiKit уже используется в проектах, то эти изменения могут потенциально что-то сломать. 

Чтобы не допустить этого есть юнит тесты, которые тестируют функционал и публичный апи компонентов. Однако, бывают только визуальные изменения, которые не меняют функциональность или публичный апи, но могут значительно изменить UI и, соответсвенно, сломать верстку в приложениях, которые их используют.

Чтобы гарантировать безопасность изменений в таких случаях можно использовать screenshot тесты, которые проверяют визуальные различия компонентов. В документации storybook есть пример таких тестов https://storybook.js.org/docs/react/writing-tests/visual-testing 

Как смотрите на то, чтобы добавить такие тесты?

@Lootjs @abdraymd

**Comment by abdraymd** [comment]
Тоже думал об этом, так что я за)
